src/main/java/uk/ac/cam/cl/dtg/isaac/api/GameboardsFacade.java

```diff
@@ -69,11 +69,13 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
-import java.util.stream.Collectors;
 
 import static com.google.common.collect.Maps.immutableEntry;
 import static uk.ac.cam.cl.dtg.isaac.api.Constants.*;
-import static uk.ac.cam.cl.dtg.segue.api.Constants.*;
```

Comment:
If you can, please leave the Constants import as a star (you can configure this in IntelliJ, for example, under "Packages to use import with "*" and adding `uk.ac.cam.cl.dtg.segue.api.Constants` and `uk.ac.cam.cl.dtg.isaac.api.Constants`).

Reply:
Ahh yes, sorry, I optimised imports and it didn't occur to me the .* was intentional